bin/check_archived_drafts.py

```diff
@@ -58,6 +58,12 @@ def main():
     all_yang_drafts_strict = os.path.join(temp_dir, 'draft-with-YANG-strict')
     missing_modules_directory = os.path.join(temp_dir, 'drafts-missing-modules')
     all_yang_path = os.path.join(temp_dir, 'YANG-ALL')
+
+    remove_all_yang_path = False
+    if not os.path.exists(all_yang_path):  # if directory exists, nothing to do
+        os.makedirs(all_yang_path)
```

Comment:
you can use `os.makedirs(all_yang_path, exist_ok=True)` instead of checking whether dir already exists